ComponentKit/LayoutComponents/CKStackLayoutComponent.h

```diff
@@ -77,10 +77,18 @@ struct CKStackLayoutComponentChild {
   CGFloat spacingBefore;
   /** Additional space to place after the component in the stacking direction. */
   CGFloat spacingAfter;
-  /** If the sum of childrens' stack dimensions is less than the minimum size, should this component grow? */
-  BOOL flexGrow;
-  /** If the sum of childrens' stack dimensions is greater than the maximum size, should this component shrink? */
-  BOOL flexShrink;
+  /**
+   If the sum of childrens' stack dimensions is less than the minimum size, how much should this component grow?
+   This value represents the "flex grow factor" and determines how much this component should be grow in relation to any
+   other flexible children.
+   */
+  NSInteger flexGrow;
+  /**
+   If the sum of childrens' stack dimensions is greater than the maximum size, how much should this component shrink?
+   This value represents the "flex shrink factor" and determines how much this component should be shrink in relation to
+   any other flexible children.
+   */
+  NSInteger flexShrink;
```

Comment:
Good point. These can likely be floats given the specification's definition of a `number`:

https://www.w3.org/TR/css3-values/#number-value